modules/ROOT/pages/rtf-k8s-practices.adoc

```diff
@@ -0,0 +1,8 @@
+== Kubernetes (K8s) Best Practices for Runtime Fabric
+
+Review the following Kubernetes best practices for Runtime Fabric. Note that to utilize the complete value of Runtime Fabric, refer to your respective Kubernetes (K8s) provider's configuration specification for details.
```

Comment:
The "Note that..., refer to" structure sounds weird. May be:
"Note that..., you can refer to...".